Can't complete ignite-cli new PizzaApp - sharp exit code 1

### Describe the bug

As part of learning the setup, executing ignite-cli new PizzaApp fails, at least on my environment. It appears to fail on the sharp execution, with an exit code of 1. I've tried with Node 20.9.0 and 22.6.0, same results in both. My environment is pretty clean. The only potential thing I can think of is I already have libvips installed via Brew.

Execution is as below:

```
npx --verbose ignite-cli@latest new PizzaApp
npm verb cli /Users/danielclarke/.nvm/versions/node/v20.9.0/bin/node /Users/danielclarke/.nvm/versions/node/v20.9.0/lib/node_modules/npm/bin/npm-cli.js
npm info using npm@10.1.0
npm info using node@v20.9.0
npm verb title npm exec ignite-cli@latest new PizzaApp
npm verb argv "exec" "--loglevel" "verbose" "--" "ignite-cli@latest" "new" "PizzaApp"
npm verb logfile logs-max:10 dir:/Users/danielclarke/.npm/_logs/2024-08-16T16_40_44_426Z-
npm verb logfile /Users/danielclarke/.npm/_logs/2024-08-16T16_40_44_426Z-debug-0.log
npm http fetch GET 200 https://registry.npmjs.org/ignite-cli 149ms (cache revalidated)
Need to install the following packages:
ignite-cli@9.8.2
Ok to proceed? (y) y
npm http fetch GET 200 https://registry.npmjs.org/ignite-cli 4ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/yaml 6ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/gluegun 10ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/deepmerge-json 12ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/sharp 11ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/pluralize 34ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/ora 36ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/colors 36ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/semver 39ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/execa 39ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/ejs 41ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/cosmiconfig 39ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/apisauce 40ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/fs-jetpack 41ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/enquirer 41ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/cli-table3 42ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/lodash.repeat 40ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/lodash.trim 41ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/yargs-parser 41ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/lodash.trimend 41ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/cross-spawn 42ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/which 44ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/lodash.camelcase 39ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/lodash.padstart 41ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/lodash.startcase 39ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/lodash.pad 44ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/lodash.padend 43ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/lodash.kebabcase 40ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/lodash.uppercase 39ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/lodash.lowercase 41ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/app-module-path 43ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/semver 38ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/tunnel-agent 37ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/node-addon-api 37ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/lodash.snakecase 41ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/prebuild-install 38ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/tar-fs 39ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/lodash.trimstart 41ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/lodash.upperfirst 40ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/lodash.lowerfirst 41ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/color 40ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/simple-get 40ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/detect-libc 39ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/object-assign 27ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/string-width 27ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/shebang-command 25ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/parse-json 27ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/axios 29ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/path-key 28ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/path-type 28ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/onetime 26ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/jake 27ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/human-signals 26ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/npm-run-path 26ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/import-fresh 30ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/merge-stream 26ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/get-stream 27ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/ansi-colors 28ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/minimatch 26ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/rimraf 27ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/chalk 24ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/is-interactive 23ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/strip-ansi 23ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/cli-cursor 24ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/strip-final-newline 30ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/isexe 23ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/wcwidth 25ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/log-symbols 24ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/cli-spinners 24ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/signal-exit 32ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/is-stream 34ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/@types%2fparse-json 37ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/lru-cache 26ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/follow-redirects 3ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/emoji-regex 1ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/is-fullwidth-code-point 2ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/parent-module 4ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/resolve-from 4ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/lines-and-columns 3ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/json-parse-even-better-errors 4ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/@babel%2fcode-frame 4ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/error-ex 5ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/shebang-regex 1ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/async 2ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/filelist 2ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/mimic-fn 3ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/brace-expansion 2ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/glob 2ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/callsites 1ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/ansi-styles 1ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/supports-color 2ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/color-convert 3ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/has-flag 3ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/color-name 0ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/balanced-match 1ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/concat-map 1ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/escape-string-regexp 3ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/ansi-regex 3ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/defaults 3ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/restore-cursor 4ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/picocolors 6ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/@babel%2fhighlight 7ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/is-arrayish 6ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/js-tokens 3ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/@babel%2fhelper-validator-identifier 4ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/once 3ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/inflight 4ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/path-is-absolute 3ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/fs.realpath 3ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/inherits 3ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/wrappy 2ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/yallist 1ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/color-string 10ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/minimist 10ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/mkdirp-classic 10ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/github-from-package 11ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/expand-template 11ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/node-abi 12ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/simple-concat 10ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/rc 11ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/napi-build-utils 13ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/bare-fs 11ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/tar-stream 11ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/pump 13ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/safe-buffer 10ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/decompress-response 13ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/bare-path 12ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/simple-swizzle 0ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/end-of-stream 4ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/ini 3ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/strip-json-comments 4ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/chownr 4ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/deep-extend 4ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/mimic-response 1ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/bare-events 8ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/bare-os 9ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/b4a 9ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/bare-stream 9ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/streamx 7ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/fast-fifo 9ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/queue-tick 1ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/text-decoder 2ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/clone 1ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/fs-constants 2ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/bl 2ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/readable-stream 2ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/buffer 4ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/util-deprecate 3ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/string_decoder 3ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/base64-js 1ms (cache hit)
npm http fetch GET 200 https://registry.npmjs.org/ieee754 2ms (cache hit)
npm WARN deprecated inflight@1.0.6: This module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and powerful.
npm WARN deprecated rimraf@2.7.1: Rimraf versions prior to v4 are no longer supported
npm WARN deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm http fetch POST 200 https://registry.npmjs.org/-/npm/v1/security/advisories/bulk 189ms
npm info run sharp@0.32.6 install node_modules/sharp (node install/libvips && node install/dll-copy && prebuild-install) || (node install/can-compile && node-gyp rebuild && node install/dll-copy)
npm info run sharp@0.32.6 install { code: 1, signal: null }
npm verb exit 1
npm verb unfinished npm timer reify 1723826447369
npm verb unfinished npm timer reify:build 1723826447852
npm verb unfinished npm timer build 1723826447852
npm verb unfinished npm timer build:deps 1723826447852
npm verb unfinished npm timer build:run:install 1723826447856
npm verb unfinished npm timer build:run:install:node_modules/sharp 1723826447856
npm verb code 1
```

### Ignite version

9.8.2

### Additional info

```
System
  platform           darwin
  arch               arm64
  cpu                12 cores     Apple M2 Max
  directory          reactlearn   /Users/danielclarke/Documents/Projects/Client/reactlearn

JavaScript (and globally-installed packages)
  node               22.6.0       /Users/danielclarke/.nvm/versions/node/v22.6.0/bin/node
  npm                10.8.2       /Users/danielclarke/.nvm/versions/node/v22.6.0/bin/npm
    corepack         0.29.3
    npm              10.8.2
  yarn               -            not installed
  pnpm               -            not installed
  bun                -            not installed
  expo               -            not installed

Ignite
  ignite-cli         9.8.2        /Users/danielclarke/.npm/_npx/bec2839915ba8fc3/node_modules/.bin/ignite
  ignite src         build        /Users/danielclarke/.npm/_npx/bec2839915ba8fc3/node_modules/ignite-cli/build

Android
  java               21.0.3       /usr/bin/java
  android home       -            undefined

iOS
  xcode              15.4
  cocoapods          Not installed

Tools
  git                git version 2.39.3 (Apple Git-146)   /usr/bin/git
```

I don't know what's changed, but the initial creation is working now. I'm using Node 22.6.0. I do have to separately install the dependencies though, with `npm install --legacy-per-deps` after it crashes out. The initial boilerplate creation errors out with this message:

```
The following error occurred:

    npm error code ERESOLVE
npm error ERESOLVE unable to resolve dependency tree
npm error
npm error While resolving: pizza-app@0.0.1
npm error Found: react@18.2.0
npm error node_modules/react
npm error   react@"18.2.0" from the root project
npm error   peer react@">=16.8.0" from @testing-library/react-native@12.6.0
npm error   node_modules/@testing-library/react-native
npm error     dev @testing-library/react-native@"^12.5.2" from the root project
npm error   1 more (react-native)
npm error
npm error Could not resolve dependency:
npm error peer react@"^18.3.1" from react-test-renderer@18.3.1
npm error node_modules/react-test-renderer
npm error   peer react-test-renderer@">=16.8.0" from @testing-library/react-native@12.6.0
npm error   node_modules/@testing-library/react-native
npm error     dev @testing-library/react-native@"^12.5.2" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.
npm error
npm error
npm error For a full report see:
npm error /Users/danielclarke/.npm/_logs/2024-08-16T20_31_30_727Z-eresolve-report.txt
npm error A complete log of this run can be found in: /Users/danielclarke/.npm/_logs/2024-08-16T20_31_30_727Z-debug-0.log
```

same here, on Node LTS:

The following error occurred:

npm error code ERESOLVE
npm error ERESOLVE unable to resolve dependency tree
npm error
npm error While resolving: cardapio-ru@0.0.1
npm error Found: react@18.2.0
npm error node_modules/react
npm error   react@"18.2.0" from the root project
npm error   peer react@">=16.8.0" from @testing-library/react-native@12.7.2
npm error   node_modules/@testing-library/react-native
npm error     dev @testing-library/react-native@"^12.5.2" from the root project
npm error   1 more (react-native)
npm error
npm error Could not resolve dependency:
npm error peer react@"^18.3.1" from react-test-renderer@18.3.1
npm error node_modules/react-test-renderer
npm error   peer react-test-renderer@">=16.8.0" from @testing-library/react-native@12.7.2
npm error   node_modules/@testing-library/react-native
npm error     dev @testing-library/react-native@"^12.5.2" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.

--- a/boilerplate/package.json
+++ b/boilerplate/package.json
@@ -90,6 +90,7 @@
     "patch-package": "^8.0.0",
     "postinstall-prepare": "1.0.1",
     "prettier": "2.8.8",
+    "react-test-renderer": "18.2.0",
     "reactotron-core-client": "^2.8.13",
     "reactotron-mst": "^3.1.7",
     "reactotron-react-js": "^3.3.11",

